deeppavlov/utils/benchmarks/benchmarks.py

```diff
@@ -0,0 +1,340 @@
+# Copyright 2017 Neural Networks and Deep Learning lab, MIPT
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+from collections import defaultdict
+from logging import getLogger
+from pathlib import Path
+from typing import Optional, Union
+
+import numpy as np
+import pandas as pd
+from tqdm import tqdm
+
+from deeppavlov import build_model
+from deeppavlov.core.commands.train import read_data_by_config, get_iterator_from_config
+from deeppavlov.core.commands.utils import parse_config, expand_path
+from deeppavlov.core.common.file import find_config, save_jsonl
+from deeppavlov.download import deep_download
+
+log = getLogger(__name__)
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument('config_path', help='path to a pipeline json config', type=str)
+parser.add_argument('benchmark_name', help='benchmark name to be submitted', choices=['glue', 'superglue', 'russian_superglue'])
+parser.add_argument('-o', '--output-file', default=None, help='path to save output', type=str)
+parser.add_argument('-d', '--download', action='store_true', help='download model components')
+
+GLUE_TASKS = {
+    'cola': 'CoLA',
+    'mnli-m': 'MNLI-m',
+    'mnli-mm': 'MNLI-mm',
+    'mrpc': 'MRPC',
+    'qnli': 'QNLI',
+    'qqp': 'QQP',
+    'rte': 'RTE',
+    'sst2': 'SST-2',
+    'stsb': 'STS-B',
+    'wnli': 'WNLI'
+}
+
+SUPER_GLUE_TASKS = {
+    'copa': 'COPA',
+    'multirc': 'MultiRC',
+    'boolq': 'BoolQ',
+    'record': 'ReCoRD',
+    'wic': 'WiC'
+}
+
+RSG_TASKS = {
+    'lidirus': 'LiDiRus',
+    'rcb': 'RCB',
+    'parus': 'PARus',
+    'muserc': 'MuSeRC',
+    'terra': 'TERRa',
+    'russe': 'RUSSE',
+    'rwsd': 'RWSD',
+    'danetqa': 'DaNetQA',
+    'rucos': 'RuCoS'
+}
+
+
+def get_predictions(model, data_gen, replace_word=None, round_res=False):
+    """Gets model predictions and replaces model output with replace_word.
+    
+    Args:
+        model: The model itself.
+        data_gen: Iterator with data to be submitted.
+        replace_word: Model outputs to be replaced with 1, other outputsare replaced with 0. If None, model outputs are not replaced.
+        round_res: If True, model outputs are rounded (used in stsb).
+    """
+
+    submission = {'index': [], 'prediction': []}
+    for idx, (x, _) in enumerate(tqdm(data_gen)):
+        prediction = model.compute(x)[0]
+        if replace_word:
+            prediction = 1 if prediction in {replace_word} else 0
```

Comment:
Replace all redundant `in`s with `==` and `!=`.

Reply:
Fixed in [commit](https://github.com/deeppavlov/DeepPavlov/pull/1647/commits/255b5c3fc5ef19d39b7ba6fb7f977402b9dc70a2)